docs/api-testing/import-postman-collection.md

```diff
@@ -123,8 +123,7 @@ Your environmental variables will now be available in the __Variables__ section
 
 Your environmental variables will now available across all your Projects.
 
-## Importing a HAR File from an RDC Job
-<p><span className="sauceDBlue">RDC Only</span></p>
+## Importing a HAR File (RDC and VDC)
 
 You can import a HAR (HTTP Archive) file into API Testing to automatically generate a functional test. Network Traffic Capture must be enabled to use this feature. See [Network Traffic Capture](/mobile-apps/features/network-capture) for more information.
```

Comment:
@lauramthomas we need to also mention this https://docs.saucelabs.com/insights/debug/#enabling-extended-debugging since it is required for VDC